app/controllers/api/orgs_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::OrgsController < ApplicationController
+  def get
+    user_with_token = User.where(access_token: params[:access_token])
```

Comment:
Make sure this is also refactored so that you get `access_token` via request headers and not `params`